docs/resources/risks/risk-score.md

```diff
@@ -170,25 +170,25 @@ We hope to improve this dimension with the help of the DeFI community to potenti
   </tr>
   <tr>
    <td>4</td>
-   <td>DD took place. Protocol contracts audited/verified. A multisig is required or contracts are Upgradable. Multisig has a low threshold of signers. No bounty program.</td>
+   <td>DD took place. Protocol contracts audited/verified. A multisig is required or contracts are upgradable. Multisig has a low threshold of signers. No bounty program.</td>
   </tr>
   <tr>
    <td>3</td>
-   <td>DD took place. Protocol contracts are audited/verified by at least <strong>one reputable audit </strong>firm. A multisig with a good threshold is required and/or contracts are immutable. Has a good bounty program.</td>
+   <td>DD took place. Protocol contracts are audited/verified by at least <strong>one reputable audit </strong>firm. A multisig with an appropriate threshold is required and/or contracts are immutable. Has a good bounty program.</td>
   </tr>
   <tr>
    <td>2</td>
-   <td>DD took place. Currently deployed protocol contracts are audited/verified by at least <strong>two reputable audit </strong>firms. A multisig with a good threshold is required and/or contracts are immutable. Has a good bounty program.</td>
+   <td>DD took place. Protocol contracts are audited/verified by at least <strong>two reputable audit </strong>firms. A multisig with an appropriate threshold is required and/or contracts are immutable. Has a good bounty program.</td>
   </tr>
   <tr>
    <td>1</td>
-   <td>Protocols involved in contracts are trusted blue chip protocols with a good record of security. For example: Maker, Uniswap, Curve, AAVE, and Compound. These protocols handle at least all the criteria specified in item 2 and more.</td>
+   <td>Protocols involved in contracts are trusted blue chip protocols with a good track record of security. For example: Maker, Uniswap, Curve, AAVE, and Compound. These protocols meet all the criteria specified in item 2 and more.</td>
   </tr>
 </table>
 
-### Team knowledge
+### Team Knowledge
 
-Measures how much expertise on a strategy is shared amongst Yearn contributors. How many contributors can manage the strategy and respond in an emergency? The less people can manage and respond during an emergency the riskier the strategy assessment in this dimension:
+Measures the amount of expertise on a strategy that is shared amongst Yearn contributors. How many contributors can manage the strategy and respond in an emergency? The less people who can manage and respond during an emergency the riskier the strategy assessment in this dimension:
```

Comment:
The fewer people who can manage and respond...

Reply:
```suggestion
Measures the amount of expertise on a strategy that is shared amongst Yearn contributors. How many contributors can manage the strategy and respond in an emergency? The fewer people who can manage and respond during an emergency the riskier the strategy assessment in this dimension:
```